docs/serving/samples/gitwebhook-go/README.md

```diff
@@ -10,6 +10,10 @@ webhook.
   and a Docker Hub account configured (we'll use it for a container registry).
 - An account on [GitHub](https://github.com) with read/write access to a
   repository.
+- A public domain name available over the internet for your service to run this
+  example, if you use the repository in github.com to send a payload to your
+  webhook service. Check the guidance on how to get one for your service
```

Comment:
Can you help add a link to the `guidance`?

Reply:
Each cloud provider should have its own guidance. I have not investigated all of them.

Reply:
make sense, thanks.

Reply:
Step 6 further down actually points to the doc about configuring a custom domain and assigning a static IP to it. I think we should bring that information to the sentence that you just added.